src/Microsoft.Azure.WebJobs.Host/Bindings/SupportsRetryAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Attribute applied to a <see cref="ITriggerBinding"/> implementation when the binding supports retry.
```

Comment:
nit: "... implementation to declare that the trigger supports cref=RetryAttribute."